File: backend-java/NewsArticlePXL/comment-service/src/main/java/pxl/be/services/domain/Comment.java
package pxl.be.services.domain;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
@Entity
@Table(name="comments")
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private Long postId;
    private String description;
    private String commenter;
    private LocalDate dateOfComment;
}

File: frontend-web/news-article-app/src/app/core/models/comments/Comment.ts
export class Comment {
  commentId: number;
  postId: number;
  description: string;
  commenter: string;
  dateOfComment: Date;

  constructor(commentId: number, postId: number, description: string, commenter: string, dateOfComment: Date) {
    this.commentId = commentId;
    this.postId = postId;
    this.description = description;
    this.commenter = commenter;
    this.dateOfComment = dateOfComment;
  }
}
